autoreact/ktp_xarray/xarray_wrappers.py

```diff
@@ -18,75 +17,106 @@ def from_data(temps, press, rates):
 
 # Getters
 def get_pressures(ktp):
```

Comment:
I have generally followed the practice of omitting the "get" prefix on functions and just naming it after the return value. So, I would prefer just to call these functions `pressures()`, `temperatures()`, etc.

Reply:
(I do add other verbs, though, such as "set", when the function isn't simply retrieving a value.)